src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4947,4 +4947,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_OutVarDeconstructionIsNotSupported" xml:space="preserve">
     <value>Deconstruction is not supported for an 'out' argument.</value>
   </data>
+  <data name="ERR_ExpressionVariableInConstructorOrFieldInitializer" xml:space="preserve">
+    <value>Out variable or pattern variable declarations are not allowed within constructor/field/auto-implemented property initializers.</value>
```

Comment:
Consider separating cases with comma for consistency with other errors. And remove `auto-implemented` if not necessary. Perhaps: `... not allowed within constructors, field initializers, or property initializers`.